webapp/portlet/src/main/webapp/vue-apps/spaces-administration/spacesAdministrationServices.js

```diff
@@ -1,13 +1,13 @@
 export function getSpaces() {
-  return fetch(`${Vue.prototype.$spacesConstants.SOCIAL_SPACE_API}?sort=date&order=desc&limit=${Vue.prototype.$spacesConstants.SPACES_PER_PAGE}&returnSize=true&expand=membersCount`, {credentials: 'include'}).then(resp => resp.json());
+  return fetch(`${Vue.prototype.$spacesConstants.SOCIAL_SPACE_API}?sort=date&order=desc&limit=${Vue.prototype.$spacesConstants.SPACES_PER_PAGE}&returnSize=true&expand=membersCount,navigations_permission`, {credentials: 'include'}).then(resp => resp.json());
```

Comment:
```suggestion
  return fetch(`${Vue.prototype.$spacesConstants.SOCIAL_SPACE_API}?sort=date&order=desc&limit=${Vue.prototype.$spacesConstants.SPACES_PER_PAGE}&returnSize=true&expand=membersCount, navigations_permission`, {credentials: 'include'}).then(resp => resp.json());
```

Reply:
it needs to be String in expand no need for space here